controllers/ovs_controller.go

```diff
@@ -299,6 +299,16 @@ func (r *OVSReconciler) reconcileNormal(ctx context.Context, instance *ovsv1beta
 		5,
 	)
 
+	if instance.Spec.TunnelNetworkCidr != "" {
+		tunnelIP, err := ovs.GetIPFromCidr(instance.Spec.TunnelNetworkCidr)
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		instance.Status.TunnelNetworkIP = tunnelIP
+	} else {
+		instance.Status.TunnelNetworkIP = "Check POD IP address"
```

Comment:
in this case I guess we want to update the status after the daemonset got created when we have the pod IPs?